.github/workflows/subset_report.yml

```diff
@@ -0,0 +1,81 @@
+name: 'Subset report'
+on:
+  pull_request:
+    branches: [ main ]
+    paths:
+      - 'src/ontology/uberon-edit.obo'
+  workflow_dispatch:
+env:
+  STATUS: 1
+jobs:
+  is_changed:
+    runs-on: macos-latest
+    # container: obolibrary/odkfull:latest
+    steps:
+      - uses: docker-practice/actions-setup-docker@master
+      - run: |
+          set -x
+          docker version
+      - name: Install ODK
+        run: docker pull obolibrary/odklite
+      # Checks-out current branch
+      - uses: actions/checkout@v2
+      # Checks-out main and prepare git diff report
+      - name: create git diff
+        id: gitdiff
+        if: ${{ github.event.pull_request.base.sha }}
+        run: |
+          git fetch origin main ${{ github.event.pull_request.base.sha }}
+          ls -l
+          git diff ${{ github.event.pull_request.base.sha }} ${{ github.sha }} src/ontology/uberon-edit.obo > gitdiff.md
+          cat gitdiff.md
+          echo ::set-output name=diff::$(cat gitdiff.md)
+      # Set Status to 0 to cancel the workflow' if keyword not in the diff file
+      - name: conditional cancel
+        if: "!contains(steps.gitdiff.outputs.diff, 'never_in_taxon')"
+        run: |
+          echo Taxon constraint updates do not exist
+          echo "STATUS=0" >> $GITHUB_ENV
+      # Checks-out main branch under "main" directory
+      - uses: actions/checkout@v2
+        if: env.STATUS == 1
+        with:
+          ref: main
+          path: main
+      - name: make step for branch
+        if: env.STATUS == 1
+        run: cd src/ontology && mkdir -p tmp reports mirror && docker run -v $PWD/../../:/work -w /work/src/ontology -e ROBOT_JAVA_ARGS='-Xmx9G' -e JAVA_OPTS='-Xmx9G' --rm obolibrary/odklite $TIMECMD make BRI=false MIR=false PAT=false IMP=false subsets/human-view.owl subsets/mouse-view.owl > TESTLOG.log
+      - name: debugging branch
+        if: ${{ failure() }}
+        run: tail -20 src/ontology/TESTLOG.log
+      - name: make step for main
+        if: env.STATUS == 1
+        run: cd main/src/ontology && mkdir -p tmp reports mirror && docker run -v $PWD/../../:/work -w /work/src/ontology -e ROBOT_JAVA_ARGS='-Xmx9G' -e JAVA_OPTS='-Xmx9G' --rm obolibrary/odklite $TIMECMD make BRI=false MIR=false PAT=false IMP=false subsets/human-view.owl subsets/mouse-view.owl > TESTLOG.log
```

Comment:
Likewise, `$TIMECMD` has no purpose here.

Reply:
Thx @gouttegd for spotting that!